conf/settings.yml

```diff
@@ -21,19 +9,25 @@ os:
         version: ["5.0", "6.0", "7.0", "8.0"]
     Linux:
     Macos:
-        version: [None, "10.6", "10.7", "10.8", "10.9", "10.10", "10.11", "10.12", "10.13", "10.14", "10.15", "11.0", "13.0"]
+        version: [None, "10.6", "10.7", "10.8", "10.9", "10.10", "10.11", "10.12", "10.13", "10.14", "10.15", "11.0", "12.0", "13.0"]
```

Comment:
Why is this conf here? I don't think it is used now, it should probably be removed too?

Reply:
I saw it this morning. This settings was used for a specific example, not migrated to conan2 yet, so yes, this should be removed